executor/agents/agno/agno_agent.py

```
@@ -651,6 +679,8 @@ def _handle_execution_error(
         Returns:
             TaskStatus: Failed status
         """
+        import os
```

Ruff (0.15.0)

[error] Redefinition of unused `os` from line 11: `os` redefined here

Remove definition: `os`

(F811)